src/group/membership/components/GroupMembersList.js

```diff
@@ -0,0 +1,256 @@
+      onSearchParamsChange={setSearchParams}
+      localeText={{
+        noRowsLabel: t('group.members_list_empty'),
+      }}
+    />
+  );
+};
+
+const MembersCards = () => {
+  const { t } = useTranslation();
+  const { members } = useContext(GroupMembersListContext);
+
+  return (
+    <List>
+      {members.map(member => (
+        <ListItem
+          key={member.email}
+          sx={{ padding: 0, marginBottom: theme.spacing(2) }}
+        >
+          <Card
+            component={Stack}
+            direction="row"
+            alignItems="center"
+            spacing={1}
+            sx={{ width: '100%', padding: theme.spacing(1) }}
+          >
+            <AccountAvatar sx={{ width: 80, height: 80 }} user={member} />
+            <Grid container spacing={2}>
+              <Grid item xs={12}>
+                <Typography variant="caption">
+                  {t('group.members_list_column_name')}
+                </Typography>
+                <Typography variant="body1">
+                  {t('user.full_name', { user: member })}
+                </Typography>
+              </Grid>
+              <Grid item xs={6}>
+                <Typography variant="caption" display="block">
+                  {t('group.members_list_column_role')}
+                </Typography>
+                <RoleSelect member={member} />
+              </Grid>
+              <Grid item xs={6}>
+                <RemoveButton member={member} />
+              </Grid>
+            </Grid>
+          </Card>
+        </ListItem>
+      ))}
+    </List>
+  );
+};
+
+const GroupMembersList = () => {
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  const isSmall = useMediaQuery(theme.breakpoints.down('md'));
+  const { list: members, fetching } = useSelector(state => state.group.members);
+
+  const { owner, groupSlug } = useGroupContext();
+
+  useEffect(() => {
+    dispatch(fetchMembers(groupSlug));
+  }, [dispatch, groupSlug]);
+
+  if (fetching) {
+    return <PageLoader />;
+  }
+
+  const onMemberRemove = member => {
+    dispatch(removeMember(member));
+  };
+
+  const onMemberRoleChange = (member, newRole) => {
+    dispatch(updateMemberRole({ member, newRole }));
+  };
+
+  const listContext = {
+    onMemberRoleChange,
+    onMemberRemove,
+    members: _.values(members),
+  };
+
+  return (
+    <GroupMembersListContext.Provider value={listContext}>
+      <Stack
+        direction="row"
+        alignItems="center"
+        justifyContent="space-between"
+        sx={{
+          marginTop: theme.spacing(2),
+          marginBottom: theme.spacing(2),
+        }}
+      >
+        <Restricted permission="group.manageMembers" resource={owner}>
+          <Typography variant="h2">
+            {t('group.members_list_title', { name: _.get(owner, 'name') })}
+          </Typography>
+        </Restricted>
+      </Stack>
+      {isSmall ? <MembersCards /> : <MembersTable />}
```

Comment:
question: should we just have one component here, i.e. `<MemberList />` and have it do the size detection? that would make it easier to reuse.

Reply:
@paulschreiber I like this, but I think this should impact the other list components, like landscape list and group list, is it ok if we create a new issue to improve this?

Reply:
Definitely. New issue is good.